misc/Kconfig

```diff
@@ -112,6 +112,13 @@ config CHECK_LINK_MAP
 	help
 	  Run a linker address generation validity checker at the end of the
 	  build.
+
+config VERSION_HEADER
+	bool "Add a version header to binary"
+	default n
+	help
+	This will add a version header section to the generated binary.
```

Comment:
2 spaces after tab

Reply:
Probably need to point to info on how-to provide content for the version header (processed by a post build script).

Reply:
ok. I will add it.